.gitignore

```diff
@@ -4,10 +4,12 @@
 /sonar/
 /.settings/
 /.idea/
+/.vscode/
 
 dependency-reduced-pom.xml
 
 .classpath
+.factorypath
```

Comment:
...

Reply:
If you make a comment, please give it a constructive description.
Not even I can infer a meaning from this.

Reply:
I didn't see the need because I imagined these changes were a nstake made while uploading files.